### PR TITLE
Add supercharger and recurring buy orders to cryptocom importer

### LINKED ISSUE #3899: crypto.com importer: missing 'Transaction Kind's
## Problem Definition

Some things I noticed with the crypto.com importer: rotkehlchen/data/importer.py

Two **'Transaction Kind's** seem to be **missing**:
* **supercharger_reward_to_app_credited**: Example line 
Timestamp (UTC) | Transaction Description | Currency | Amount | To Currency | To Amount | Native Currency | Native Amount | Native Amount (in USD) | Transaction Kind
2021-10-28 03:16:37 | Supercharger Reward | AXS | 0.00356292 |   |   | EUR | 0.38 | 0.4430698990794 | supercharger_reward_to_app_credited
* **recurring_buy_order**: probably needs to be handled as Trade: https://github.com/rotki/rotki/blob/develop/rotkehlchen/data/importer.py#L240

Not an issue but maybe worth to mention it: I don't think there is a 'Transaction Kind' reinbursment https://github.com/rotki/rotki/blob/develop/rotkehlchen/data/importer.py#L326. reimbursement does exist though and already is handled as Trade in https://github.com/rotki/rotki/blob/develop/rotkehlchen/data/importer.py#L248

Rotki version: 1.23.0

### PR DESCRIPTION
**Pull request by yabirgb**
Closes #3899 

Also adds a mapping for AXS in crypto.com

### CODE REVIEW

File: rotkehlchen/assets/converters.py [hunk 1]
```diff
@@ -1031,3 +1033,18 @@ def asset_from_bitpanda(bitpanda_name: str) -> Asset:
 
     our_name = BITPANDA_TO_WORLD.get(bitpanda_name, bitpanda_name)
     return symbol_to_asset_or_token(our_name)
+
+
+def asset_from_cryptocom(crtptocom_name: str) -> Asset:
```
Comment (LefterisJP) [inline rotkehlchen/assets/converters.py:1038]:
```suggestion
def asset_from_cryptocom(cryptocom_name: str) -> Asset:
```

giphy (2)

**Review by LefterisJP — COMMENTED** [review]
Have you made a binary to give to the user to see this works? If not make sure to tell me after this is merged so we can start one and  give one to him and also to the Mojave guys to confirm fixes.

**Comment by yabirgb** [comment]
Opppsssssssss fixed @LefterisJP. I didn't make any binary for this, just used the entries provided in the tests

**Review by LefterisJP — APPROVED** [review]
LGTM!

**Pull request merged**